readme.md

```diff
@@ -1388,6 +1389,21 @@ export default () => <div>
 </div>
 ```
 
+### Starting a dev server on alternative IP
+
+To start the dev server using a hostname different from the defualt `localhost` you can use the provided `--hostname or -H` option with `next dev`.
+
+```bash
+next dev <dir> -H <hostname>
```

Comment:
I believe this could be made redundant when you change above `--hostname or -H` to `--hostname hostname_here` or `-H hostname_here`

Reply:
Meaning this line can be removed after changing that.